benchmarks/nel/README.md

```diff
@@ -49,20 +47,9 @@ in the project directory.
 
 | File | Source | Description |
 | --- | --- | --- |
-| `assets/wiki/wikidata_entity_dump.json.bz2` | URL | Wikidata entity dump. Download can take a long time! |
-| `assets/wiki/wikipedia_dump.xml.bz2` | URL | Wikipedia dump. Download can take a long time! |
-| `assets/wiki/wikidata_entity_dump_filtered.json.bz2` | URL | Filtered Wikidata entity dump for demo purposes. |
-| `assets/wiki/wikipedia_dump_filtered.xml.bz2` | URL | Filtered Wikipedia dump for demo purposes. |
+| `assets/reddit.zip` | URL | Entity linking dataset scraped from Reddit. See [paper](https://arxiv.org/abs/2101.01228). |
```

Comment:
What is the status of these edits? (This doesn't seem like it's just formatting?)

Reply:
Ah, thanks for catching this. This is indeed not intentional.

Reply:
Reverted non-formatting changes in https://github.com/explosion/projects/pull/121/commits/7de339bc19d82deda49f70014debabe252fed9a7.